cmd/abigen/namefilter.go

```diff
@@ -7,7 +7,6 @@
 // original code from which it is derived.
 //
 // Much love to the original authors for their work.
-// **********
```

Comment:
Why the removal? The geth files seem to have no copyright at all, so I'm not sure why we would modify this 🤔

Reply:
to match coreth's libevm branch.

Reply:
Don't we want to match geth more than coreth/subnet-evm ultimately? 🤔